tools/lint/commands.c

```diff
@@ -58,7 +58,7 @@ cmd_clear_help(void)
 void
 cmd_print_help(void)
 {
-    printf("print [-f (yang | yin | tree [<tree-options>] | info [-P <info-path>] | jsons)] [-o <output-file>]"
+    printf("print [-f (yang | yin | tree [<tree-options>] | info [-P <info-path>] [-(-s)ingle-node])] [-o <output-file>]"
```

Comment:
I remember that in the master `-s` means `--strict`
I don't think change the user's habit is a good way.

Reply:
But there it is for parsing, this is printing functionality. Strict in printing have no meanings.

Reply:
Exactly. However, I think  the upper case `-S` may be better.